Sources/Async/EventLoop/Epoll/EpollEventLoop.swift

```diff
@@ -70,9 +70,12 @@ public final class EpollEventLoop: EventLoop {
             let event = eventlist[i]
             let source = event.data.ptr.assumingMemoryBound(to: EpollEventSource.self).pointee
 
-            if event.events & EPOLLERR.rawValue > 0 {
-                let reason = String(cString: strerror(Int32(event.data.u32)))
-                fatalError("An error occured during an event: \(reason)")
+            guard event.events & EPOLLERR.rawValue == 0 else {
+                var error: Int32 = 0
+                var errlen = socklen_t(4)
+                getsockopt(source.descriptor, SOL_SOCKET, SO_ERROR, &error, &errlen)
```

Comment:
We should probably have a `source.onError` or something for delivering this in a way that doesn't interrupt the event loop. Maybe just comment that out (useful code) and put a FIXME here?